src/py42/clients/cases.py

```diff
@@ -0,0 +1,196 @@
+from datetime import datetime
+
+from py42.util import get_attribute_keys_from_class
+from py42.util import parse_timestamp_to_milliseconds_precision
+
+
+class CaseStatus(object):
+    OPEN = u"OPEN"
+    CLOSED = u"CLOSED"
+
+    @staticmethod
+    def choices():
+        return get_attribute_keys_from_class(CaseStatus)
+
+
+class Py42InvalidCaseStatus(Exception):
```

Comment:
Custom error classes should end in the word `Error`

Reply:
aah, I always miss this. I am on it.

Reply:
Removed.